build-system/babel-plugins/babel-plugin-transform-html-template/test/fixtures/transform-assertions/indirect/input.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const x = html;
+console.log(x`html usage`);
```

Comment:
Indirect usage is actually forbidden by the linter.

Reply:
Added this test to ensure the transform didn't accidentally pick these up in case the linter fails for some reason.